docs/configure

```diff
@@ -478,10 +480,11 @@ ditch_maxmsgsize:
 		and domain = '${quote_mysql:$domain}' \
 		and users.maxmsgsize > 0 \
 		and users.domain_id=domains.domain_id}{${value}K}fail} Kb.
-  local_part_suffix = -*
-  local_part_suffix_optional
+#  local_part_suffix = -*
+#  local_part_suffix_optional
```

Comment:
Ok, let's keep the local_part_suffix in the configuration, I will go through the conf.d/-files and add these.

Is + the more common suffix? Or we use both?
````
local_part_suffix = +* : -*
````